scripts/run-integration-tests.sh

```diff
@@ -0,0 +1,24 @@
+set -e
```

Comment:
If we don't set this and any script fails, but the last one succeeds, the script exists with 0 code and is considered a success by the action